### PR TITLE
feat(multiselect): Bug fix and large size. - FRONT-3178

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

**Review by papegaill — COMMENTED** [review]
We have small and large sizes, but what about normal size(medium). 
Same as the form input, it seems that this default size is missing, 
or maybe a control issue?

**Review by papegaill — CHANGES_REQUESTED** [review]
Icon display issue for the large version when no option within the field:
Screenshot 2021-09-27 at 12 11 45
Screenshot 2021-09-27 at 12 11 37
To me icon can be a bit smaller ;)

**Comment by papegaill** [comment]
> @papegaill I actually found a problem here, we don't actually have the medium size, the normal size of the multiselect is the sm size of the select, so we need to change the default multiselect size to something bigger, and keep the default as the small size.

Thks, can we avoid using `form-control-md` class as it doesn't seems to be part of boostrap? 
https://getbootstrap.com/docs/4.0/components/forms/#sizing
Can we keep same approach as all forms elements?

I also noticed previously that all the form elements on the search page and maybe other ones are on small sizes, not sure this is correct, maybe we will have to change everything to default(medium) sizes, to confirm.. if yes, will ba another ticket.

**Comment by tibi2303** [comment]
@papegaill thanks for the remarks, we need to confirm the sizes from search and user pages with the designers, anyway i removed the md size. 👍

File: src/components/bcl-select/select.html.twig [hunk 1]
```diff
@@ -37,12 +37,14 @@
 {% set _label_class = 'form-label' %}
 {% set _hidden_label = hidden_label|default(false) %}
 
-{% set _classes = _classes|merge(['form-select-' ~ _size]) %}
-
 {% if _clean_class %}
   {% set _classes = [] %}
 {% endif %}
 
+{% if _size is not empty %}
```
Comment (planctus) [inline src/components/bcl-select/select.html.twig:44]: mmh..i believe what we are doing here is going to conflict with what we will need later...the behavior of these form elements in terms of their size is not anymore left to the user choosing a size for it and it should actually be handled via css media queries basing on the viewport.
What has been said is that the size would be "lg" in mobile viewport and "md" in the others as far as i remember

File: src/themes/default/src/scss/_multiselect.scss [hunk 2]
```diff
@@ -26,4 +27,83 @@
...
+  }
+  .ss-values {
+    .ss-value {
+      font-size: 1rem;
+    }
+    .ss-disabled {
+      padding: 8px 5px;
+    }
+  }
+  &.form-select-sm {
```
Comment (planctus) [inline src/themes/default/src/scss/_multiselect.scss:51]: basically this adaptations should not be based on the class but on a media query, if i'm correct

**Review by planctus — COMMENTED** [review]
I believe the "styles" are good, but the approach needs probably to be changed basing on this:
https://citnet.tech.ec.europa.eu/CITnet/confluence/display/FRONT/BCL+updates+to+align+on+OEL

File: src/components/bcl-select/select.html.twig (hunk 1 above)
Comment (tibi2303) [reply src/components/bcl-select/select.html.twig:44]: done. hope it's good

File: src/themes/default/src/scss/_multiselect.scss (hunk 2 above)
Comment (tibi2303) [reply src/themes/default/src/scss/_multiselect.scss]: done.

**Comment by planctus** [comment]
I'm confused, think we do too many things not in the right order, in any case since we are dealing with these sizes for all the form elements i think you can already remove everything related to the size from the template and the story

**Comment by tibi2303** [comment]
@planctus I agree, the good order should be : 
1. https://github.com/openeuropa/bootstrap-component-library/pull/81 (sizes) - I already remove the sizes you said in this pr.
2. This pr
3. Search form

**Review by planctus — APPROVED** [review]

**Pull request merged**